sagify/commands/cloud.py

```diff
@@ -84,14 +84,16 @@ def upload_data(dir, input_dir, s3_dir):
     help="time-out in seconds (default: 24 * 60 * 60)"
 )
 @click.option(
-    u"-t", u"--tags",
+    u"-t", u"--aws-tags",
```

Comment:
The short alias of `aws-tags` and `docker-tags` is the same. They need to be disambiguated.

Reply:
@pm3310 Ok, what suggestions do you have for an improved short alias? I thought that it would be ok to leave the short options the same, since there isn't any internal conflict.

I'd like to leave `-t` as the short form for `--docker-tags`, since this would leave it identical to the `-t` option for `docker build`, and this is what happens under the hood.

However, this could be changed to `-d` for docker. Maybe `--aws-tags` could then be changed to `-a` for aws. Thoughts?

Reply:
@berlinquin How about `-a` for `--aws-tags` and `-t` for `--docker-tag`? `-d` is reserved for `--dir`, otherwise it would be great.